### PR TITLE
UG: Add more description for URL.

### PR DESCRIPTION
**Pull request by chwoozy**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -677,7 +677,7 @@ This glossary provides definitions for the special terms used in this user guide
 
 **Recurring Session:** Repeating session that occurs every number of days.
 
-**URL:** The address of a web page.
+**URL:**  Uniform Resource Locator, is an address of a web page. e.g. **https://www.google.com/**
```
Comment (enhao25) [inline docs/UserGuide.md]: Haha honestly I think putting Uniform resource locator is even more confusing to ppl. Although putting the example is a good idea ah.

**Review by enhao25 — COMMENTED** [review]
One comment

File: docs/UserGuide.md (hunk 1 above)
Comment (chwoozy) [reply docs/UserGuide.md]: Haha wait but that's how the acronym URL came about.

Comment (chwoozy) [reply docs/UserGuide.md]: Okay amended god hao!

**Review by enhao25 — APPROVED** [review]
LGTM

**Pull request merged**